libs/data-access/migrations/1706720380014-migrate-paused-status-to-archived.ts

```diff
@@ -0,0 +1,83 @@
+import type { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migratePausedStatusToArchived1706720380014 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      -- Change all support status from paused innovations to closed and update the archive_snapshot
+      UPDATE innovation_support
+          SET status = 'CLOSED', archive_snapshot = JSON_MODIFY(
+          JSON_MODIFY(
+              JSON_MODIFY(
+                  COALESCE(archive_snapshot, '{}'),
+                  '$.archivedAt', CONVERT(VARCHAR, i.status_updated_at)
+              ),
+              '$.assignedAccessors', JSON_QUERY('[]')
+          ),
+          '$.status', (
+              CASE
+                  WHEN s.status = 'UNASSIGNED' THEN 'ENGAGING'
+                  ELSE s.status
+              END
+              )
+          )
+          FROM innovation_support s
+          INNER JOIN innovation i ON i.id = s.innovation_id
+          WHERE i.status = 'PAUSED' AND s.[status] != 'CLOSED';
+
+      -- Update all paused innovations to archived
+      UPDATE innovation
+          SET status = 'ARCHIVED'
+          WHERE status = 'PAUSED';
+
+      -- Remove PAUSED from the innovation status constraint
+      ALTER TABLE "innovation" DROP CONSTRAINT "CK_innovation_status";
+      ALTER TABLE "innovation" ADD CONSTRAINT "CK_innovation_status" CHECK ([status] IN (
+          'CREATED',
+          'WAITING_NEEDS_ASSESSMENT',
+          'NEEDS_ASSESSMENT',
+          'IN_PROGRESS',
+          'WITHDRAWN',
+          'COMPLETE',
+          'ARCHIVED'
+      ));
+    `);
+
+    // Remove PAUSED status and add ARCHIVED
+    await queryRunner.query(`
+    CREATE OR ALTER VIEW [innovation_grouped_status_view_entity] AS
+      SELECT innovation.id,
+          CASE
+              WHEN innovation.status IN ('CREATED') THEN 'RECORD_NOT_SHARED' -- Removed Paused
```

Comment:
Maybe here you could just do liek the other ones: `WHEN innovation.status = 'CREATED' THEN 'CREATED'`

Reply:
Great catch! Just changed in the latest commit!